### DISCUSSION TITLE
How to access request headers in subscriptions?

### DISCUSSION THREAD

**Discussion by benkeil** [Netflix/dgs-framework#442] [Q&A]
`DgsCustomContextBuilderWithRequest.build` gets called but without any data, how to access request headers?

**Comment by paulbakker** [comment]
Websockets don't have headers, which is why this isn't available.

**Reply by benkeil** [reply]
Is there any other way to get the same behaviour like querying? I need access to the bearer token.

**Reply by ghost** [reply]
https://github.com/Netflix/dgs-framework/issues/450
